SpringBootRestWS/src/main/java/com/example/Entity/Person.java

```diff
@@ -0,0 +1,93 @@
+package com.example.Entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Person {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@Column(name = "id")
+	private Long id;
+
+	public Person(Long id) {
+		this.id = id;
+	}
```

Comment:
Be better create this constructor after private variables and near of other constructors